### PR TITLE
CLI performance improvement changes & optimized BGP RIB code 

### PR DESCRIPTION
**Pull request by venkatmahalingam**
UT:
sonic# show ip bgp summary
BGP router identifier 20.0.0.1, local AS number 200
Neighbor        V   AS    MsgRcvd   MsgSent   InQ     OutQ    Up/Down                 State/PfxRcd
Ethernet8       4   900   0         0         0       0       never                   Idle
Vlan1001        4   100   0         0         0       0       never                   Idle
20.0.0.2        4   100   336       82        0       0       1d03h44m                10004
sonic# show ip bgp ipv6 summary
BGP router identifier 20.0.0.1, local AS number 200
Neighbor        V   AS    MsgRcvd   MsgSent   InQ     OutQ    Up/Down                 State/PfxRcd
2222::2         4   100   27        26        0       0       1d07h20m                55
sonic#

### CODE REVIEW

File: src/CLI/clitree/cli-xml/ipv4-neighbors.xml [hunk 1]
```diff
@@ -62,29 +62,161 @@ limitations under the License.
...
+                        optional="true">
+                    </PARAM>
                 </PARAM>
             </PARAM>
+            <PARAM 
+                name="Loopback" 
+                help="Loopback Interface Configuration" 
+                mode="subcommand" 
+                ptype="SUBCOMMAND">
+                <PARAM
```
Comment (venkatmahalingam) [inline src/CLI/clitree/cli-xml/ipv4-neighbors.xml:93]: please remove loopback, mgmt and vxlan intfs in all places.

File: src/translib/transformer/xfmr_bgp_nbr.go [hunk 2]
```diff
@@ -33,11 +34,106 @@ func init () {
...
+            if (cmd_err != nil) {
+                log.Errorf ("Failed !! Error:%s", cmd_err);
+                return tblList, err
+            }
+
+            if outError, ok := bgpNeighOutputJson["warning"] ; ok {
+                log.Errorf ("%s failed !!, %s", outError)
+                return tblList, err
+            }
+
```
Comment (venkatmahalingam) [inline src/translib/transformer/xfmr_bgp_nbr.go:121]: Ashok - As discussed this wont work when we have the static nbr only in the DB but not in FRR, can you please change the logic to loop nbrs from config DB and dynamic nbrs from FRR output.

File: src/translib/transformer/xfmr_bgp_nbr.go [hunk 3]
```diff
@@ -677,6 +773,11 @@ func fill_nbr_state_timers_info (nbr_key *_xfmr_bgp_nbr_state_key, frrNbrDataVal
         nbrTimersState.NegotiatedHoldTime = &_neg_hold_time
     }
 
+    if value, ok := frrNbrDataJson["bgpTimerKeepAliveIntervalMsecs"] ; ok {
+        _keepaliveInterval := (value.(float64))/1000
+        nbrTimersState.KeepaliveInterval = &_keepaliveInterval
+    }
+
```
Comment (venkatmahalingam) [inline src/translib/transformer/xfmr_bgp_nbr.go:884]: @rathnasabapathyv can you please look few missing state fields and add handling.

Comment (rathnasabapathyv) [reply src/translib/transformer/xfmr_bgp_nbr.go:884]: Yes after reviewing that, i have seen quite few nbr/state attributes are missing.. Those are the one's that we have introduced as bgp-ext annotation post subtree transformer implemetation for nbr/state. Added support for these attributes now

File: src/CLI/clitree/cli-xml/ipv4-neighbors.xml (hunk 1 above)
Comment (bhavini-gada) [reply src/CLI/clitree/cli-xml/ipv4-neighbors.xml:93]: @stalabi1  - Is this done ?

Comment (stalabi1) [reply src/CLI/clitree/cli-xml/ipv4-neighbors.xml:93]: yes @bhavini-gada

File: src/translib/transformer/xfmr_bgp_nbr.go (hunk 2 above)
Comment (venkatmahalingam) [reply src/translib/transformer/xfmr_bgp_nbr.go:121]: This issue has been addressed by Ashok.

**Review by dutta-partha — APPROVED** [review]

File: src/translib/transformer/xfmr_bgp_nbr.go (hunk 2 above)
Comment (AshokDaparthi) [reply src/translib/transformer/xfmr_bgp_nbr.go:121]: Done

**Review by prvattem — APPROVED** [review]

**Pull request merged**